File: CodingBat/Python/Warmup-2/array_front9.py
"""
Given an array of ints, return True if one of the first 4 elements in the array is a 9. The array length may be less than 4.


array_front9([1, 2, 9, 3, 4]) → True
array_front9([1, 2, 3, 4, 9]) → False
array_front9([1, 2, 3, 4, 5]) → False

@author unobatbayar

"""
def array_front9(nums):
  for x in range(0, len(nums)): #search for 9
    if nums[x] == 9 and x < 4:
      return True
  return False

File: CodingBat/Java/Warmup-2/arrayFront9.java
import java.util.*;
/**
 
Given an array of ints, return true if one of the first 4 elements in the array is a 9. The array length may be less than 4.


arrayFront9([1, 2, 9, 3, 4]) → true
arrayFront9([1, 2, 3, 4, 9]) → false
arrayFront9([1, 2, 3, 4, 5]) → false
  
@author unobatbayar   
 */

class arrayFront9{
    public boolean arrayFront9(int[] nums) {
        int end = nums.length;
        if(end >4) end = 4;
        
        for (int i=0; i<end; i++)
        {
          if(nums[i] == 9) return true;
        }
        return false;
      }
      

}